modules/hcp-openstack-accessing.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-manage/hcp-manage-openstack.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-openstack-accessing_{context}"]
+= Accessing the hosted cluster
+
+You can access hosted clusters on {rh-openstack-first} by either getting the `kubeconfig` file and `kubeadmin`
+credential directly from
+resources, or by using the `hcp` command-line interface to generate a `kubeconfig` file.
+
+.Prerequisites
+
+To access the hosted cluster by getting the `kubeconfig` file and credentials directly from resources, you must be familiar with the access secrets for hosted clusters. The _hosted cluster (hosting)_ namespace contains hosted cluster resources and the access secrets. The _hosted control plane_ namespace is where the hosted control plane runs.
+
+The secret name formats are as follows:
+
+** `kubeconfig` secret: `<hosted_cluster_namespace>-<name>-admin-kubeconfig`. For example, `clusters-hypershift-demo-admin-kubeconfig`.
+** `kubeadmin` password secret: `<hosted_cluster_namespace>-<name>-kubeadmin-password`. For example, `clusters-hypershift-demo-kubeadmin-password`.
+
+The `kubeconfig` secret contains a Base64-encoded `kubeconfig` field, which you can decode and save into a file to use with the following command:
```

Comment:
This is confusing as it suggests the following command demonstrates how to decode (the kubeconfig) and save into a file for use (with other commands).

```suggestion
The `kubeconfig` secret contains a Base64-encoded `kubeconfig` field, which you can decode and save into a file. Once saved, you can the kubeconfig file to access the hosted cluster:
```

Alternatively, you could demonstrate how to decode the secret. Something like the following:

```
oc get secret <hosted_cluster_namespace>-<name>-admin-kubeconfig -o go-template='{{ .data.kubeconfig | b64dec }} > <hosted_cluster_namespace>-<name>.kubeconfig'
```

Reply:
Later: you actually give a better example [below using the `oc extract` command](https://github.com/openshift/openshift-docs/pull/93726/files#diff-ce5379c23e099958374fc692ccdd8efaa91b1da29952a2be934611c1536602a3R75). Maybe it's worth repeating that here?

Reply:
Yeah, sorry. That's all boilerplate from the general HCP docs collection. I'll take a swing at fixing it for this context, though.